lib/transforms/addDataVersionAttributeToHtmlElement.js

```diff
@@ -1,20 +1,17 @@
 const childProcess = require('child_process');
-const Promise = require('bluebird');
 
 module.exports = (queryObj, version) => {
   return async function addDataVersionAttributeToHtmlElement(assetGraph) {
     if (typeof version === 'undefined') {
       // Try to derive a version tag from git:
       try {
-        const [stdout] = await Promise.fromNode(
-          cb =>
-            childProcess.exec(
-              'git describe --long --tags --always --dirty',
-              cb
-            ),
-          { multiArgs: true }
+        const rawVersion = childProcess.execSync(
```

Comment:
Ouch, could we avoid `execSync`? Maybe something like:

```diff
diff --git a/lib/transforms/addDataVersionAttributeToHtmlElement.js b/lib/transforms/addDataVersionAttributeToHtmlElement.js
index 3debad62..bf958461 100644
--- a/lib/transforms/addDataVersionAttributeToHtmlElement.js
+++ b/lib/transforms/addDataVersionAttributeToHtmlElement.js
@@ -1,16 +1,20 @@
 const childProcess = require('child_process');
+const { promisify } = require('util');
 
 module.exports = (queryObj, version) => {
   return async function addDataVersionAttributeToHtmlElement(assetGraph) {
     if (typeof version === 'undefined') {
       // Try to derive a version tag from git:
       try {
-        const rawVersion = childProcess.execSync(
-          'git describe --long --tags --always --dirty',
-          {
-            encoding: 'utf-8'
-          }
-        );
+        const rawVersion = await promisify(cb =>
+          childProcess.exec(
+            'git describe --long --tags --always --dirty',
+            {
+              encoding: 'utf-8'
+            },
+            cb
+          )
+        )();
         version = rawVersion.replace(/^[\s\n\r]+|[\s\r\n]+$/g, '');
       } catch (e) {}
     }
```

Reply:
Fixed in 43a1ae24

Why doesn't this work?

```js
const rawVersion = await promisify(childProcess.exec)(
  'git describe --long --tags --always --dirty',
  {
    encoding: 'utf-8'
  }
);
```

Reply:
Hmm, yeah, that's more elegant than involving a callback. It does work if you destructure the fulfillment value:

```js
        const { stdout } = await promisify(childProcess.exec)(
          'git describe --long --tags --always --dirty',
          {
            encoding: 'utf-8'
          }
        );
        version = stdout.replace(/^[\s\n\r]+|[\s\r\n]+$/g, '');
```

Reply:
Ah! I'll make that change.

Reply:
Done in b810eab7